pkg/kancontroller/kancontroller.go

```diff
@@ -112,3 +120,17 @@ func isCACertMounted() bool {
 
 	return true
 }
+
+func createOrUpdateCRDs() bool {
+	createOrUpdateCRD := os.Getenv(createOrUpdateCRDEnvVar)
+	if createOrUpdateCRD == "" {
+		return false
```

Comment:
Shouldn't this be true by default? What happens to the current installations where the CRDs are handled by the controller?

Reply:
we decided to use the default to be `false` because we want CRDs to be primarily managed/installed by helm instead of kanister controller.
if the CRDs are already installed by  controller and we try to upgrade that release it is going to go through and release would be upgraded without any problem.

Reply:
how much impact will this default have on existing users, who are currently relying on the controller to manage the CRDs? with this change, these users need to be made aware that by default, they will be responsible for manually upgrading the CRDs going forward because Helm doesn't do it for them, unless they change this configuration to `true` in all future upgrades.

Reply:
Should we document this @ihcsim ?

Reply:
Per discussion in community meeting, we will keep the controller-managed approach as the default, to avoid disruption to existing upgrade path. User can use Helm to manage the CRDs by changing the `controller.updateCRDs` configuration to `false`.

Reply:
I have  made the changes, by default controller is going to be responsible for managing CRDs.